test/auto_test_almond.js

```diff
@@ -3736,7 +3736,7 @@ async function main(limit = Infinity) {
                 ret[k] = {type: 'oauth2', kind: 'com.instagram', text: 'Instagram'};
             else if (k === 'org.thingpedia.rss')
                 ret[k] = _rssFactory;
-            else if (k === 'org.thingpedia.builtin.thingengine.home' || k === 'car')
+            else if (k === 'org.thingpedia.builtin.thingengine.home' || k === 'org.thingpedia.builtin.thingpedia.morning' || k === 'car')
```

Comment:
Mh? What's this about?
"org.thingpedia.builtin.thingengine.home" is a reference to Almond Home, an experimental Thingpedia device that would behave like a smart speaker (like a Google Home). It has nothing to do with home locations.